README.md

```diff
@@ -28,12 +28,28 @@ Die [Koalitionsvereinbarung für die Stadtratsperiode 2020 –2026 zwischen Ober
 Dieses Konzept definiert [Free Open Source Software](https://de.wikipedia.org/wiki/Free/Libre_Open_Source_Software) (FOSS) für die Landeshauptstadt München auf Basis des Koalitionsvertrags.
 Es definiert Voraussetzungen, Abhängigkeiten, Ausschlüsse und Zielrichtung von Entwicklung, Beschaffung und Betrieb von OpenSource Software.
 
+## Offene Standards als Basis
 
-## Offene Standards
+## Verwendung von FOSS in der LHM
```

Comment:
Ich finde, "Verwendung" könnte implizieren, dass nur das schon verwendete beschrieben wird. Wir wollen ja aber weiter gehen und auch darstellen, was in Zukunft verwendet werden SOLL.